src/OptPass.cc

```diff
@@ -60,9 +71,134 @@ bool OptPass::runOnModule(Module&)
 
   outs() << "Got policy file!\n";
 
-  return false;
-}
+	std::map<CallInst*, std::vector<Policy::Direction>> callInstsMap;
```

Comment:
This whitespace is inconsistent with the rest of the function: the [LLVM style guide](http://llvm.org/docs/CodingStandards.html#use-spaces-instead-of-tabs) dictates the use of spaces instead of tabs.